VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -143,52 +144,79 @@ export class VideoCallService {
 
     updateCameraForCall(camera: UserMediaDevice) {
         this.pexipAPI.video_source = camera.deviceId;
-        this.logger.info(`[VideoCallService] - Using preferred camera: ${camera.label}`);
+        this.logger.info(`${this.loggerPrefix}  Using preferred camera: ${camera.label}`);
     }
 
     updateMicrophoneForCall(microphone: UserMediaDevice) {
         this.pexipAPI.audio_source = microphone.deviceId;
-        this.logger.info(`[VideoCallService] - Using preferred microphone: ${microphone.label}`);
+        this.logger.info(`${this.loggerPrefix} Using preferred microphone: ${microphone.label}`);
     }
 
-    toggleMute(): boolean {
+    toggleMute(conferenceId: string, participantId): boolean {
+        this.logger.info(`${this.loggerPrefix} Toggling mute`, {
+            currentMuteStatus: this.pexipAPI.mutedAudio,
+            currentVideoStatus: this.pexipAPI.mutedVideo,
+            conference: conferenceId,
+            participant: participantId
+        });
         return this.pexipAPI.muteAudio();
```

Comment:
log after too maybe? if so a couple in this file

Reply:
The callback in participants panel component (see line 112) will capture when this is successful.